cpp/ql/src/semmle/code/cpp/dataflow/internal/FlowVar.qll

```diff
@@ -427,7 +431,7 @@ module FlowVar_internal {
     /**
      * Gets a variable that is assigned in this loop and read outside the loop.
      */
-    private Variable getARelevantVariable() {
+    Variable getARelevantVariable() {
```

Comment:
I suspect removing `private` here was a debugging thing that slipped into the commit, right?

Reply:
No, I'm using that for some manual magic further down.

Reply:
Oh, right!